.changes/1232-chat-media-improvements.md

```diff
@@ -0,0 +1,4 @@
+- [Improvement] : Now have better control on chat media download. Chat media like images, videos, files etc will now download automatically. User can download media based on his/her choice.
```

Comment:
By downloading automatically, you mean thumbnails? If so, please mention here to avoid confusion.

Reply:
Oppos!, a typo mistake here.
- - - 
❌ ~~Chat media like images, videos, files, etc will now download automatically.~~
✅ Chat media like images, videos, files, etc will not download automatically.